### PR TITLE
trying singularityce again

### PR DESCRIPTION
**Pull request by ngam**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ngam** [comment]
@conda-forge/staged-recipes this is ready to go (only relevant build is on linux_64_centos7)

Please note and provide feedback:
1. This is simply a copy with minor edits of the singularity-feedstock (https://github.com/conda-forge/singularity-feedstock). Sylabs (SingularityCE) forked HPCng's Singularity in May and now they're starting to diverge (confusingly). SingularityCE is backed by a commercial entity (while still open source) and thus their dev seems more focused and speedy. I thought we should add the CE fork feedstock to keep track of the changes and provide both the original and the fork. I am a maintainer on the singularity-feedstock, and I have added a conflict in the recipe here.
2. I am not sure if we have preferences about using singularityce vs singularity-ce (they weirdly use them interchangeably, so I am not sure what you prefer/think is best)
3. I disabled tests that require `--privileged` flag and will reenable those later when this becomes a feedstock
4. I simply more or less copied from the singularity-feedstock --- is this okay? Do I need to rewrite stuff or like cite or something?

**Comment by ngam** [comment]
minor edits to bring macos up to speed; and now linux64 cos6 works fine.

**Comment by ngam** [comment]
@conda-forge/staged-recipes gentle reminder to review this when you have a chance :)

File: recipes/singularityce/build.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  -x "${CXX}"
+
+# build
+pushd builddir
+export LD_LIBRARY_PATH=${PREFIX}/lib
+make
+
+# install
+make install
+
```
Comment (ocefpaf) [inline recipes/singularityce/build.sh:31]: Is there a `make check` we can run here?

Comment (ngam) [reply recipes/singularityce/build.sh:31]: Hi @ocefpaf! I don't think so... or at least I haven't seen it. 

For reference, 

1. this is our original singularity here at conda-forge: [link](https://github.com/conda-forge/singularity-feedstock/blob/7ec89a8b71fa136d66f46367101890fcd2c1990f/recipe/build.sh#L24-L30)
2. and this is the dev install docs: [link](https://sylabs.io/guides/latest/admin-guide/installation.html#:~:text=%24%20./mconfig%20%2D%2Dwithout%2Dsuid%20%2D%2Dprefix%3D/home/dave/singularity%2Dce%20%26%26%20%5C%0A%20%20%20%20make%20%2DC%20./builddir%20%26%26%20%5C%0A%20%20%20%20make%20%2DC%20./builddir%20install)

Do you think we should be adding a `make check` here as well?

Comment (ocefpaf) [reply recipes/singularityce/build.sh:31]: If it is available, yes. I'll leave that to you in the feedstock.

**Pull request merged**